Bottom: 61bb8e31bc7ec14d352ba45da31568d51ba5d812
Top:    8c4e8595773517a9e21fcd7b76989bb8995d3e3a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-27 17:17:25 +0000

Refresh of hid-gtk-clean-up-mouse-warping

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 3bb2321..991c97d 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -497,12 +497,6 @@ ghid_set_crosshair (int x, int y, int action)
        * but the value we've been given is relative to your drawing area
        */
       gdk_window_get_origin (gport->drawing_area->window, &xofs, &yofs);
-
-      /*
-       * Note that under X11, gdk_display_warp_pointer is just a wrapper around XWarpPointer, but
-       * hopefully by avoiding the direct call to an X function we might still work under windows
-       * and other non-X11 based gdk's
-       */
       gdk_display_warp_pointer (display, screen, xofs + Vx (x), yofs + Vy (y));
     }
 }
@@ -1704,12 +1698,6 @@ Center(int argc, char **argv, int x, int y)
   gdk_display_get_pointer (display, NULL, &cx, &cy, NULL);
   gdk_display_warp_pointer (display, screen, cx - dx, cy - dy);
 
-  /*
-   * Note that under X11, gdk_display_warp_pointer is just a wrapper around XWarpPointer, but
-   * hopefully by avoiding the direct call to an X function we might still work under windows
-   * and other non-X11 based gdk's
-   */
-
   return 0;
 }
